core-bundle/src/Controller/FrontendModule/RootPageDependentModulesController.php

```diff
@@ -31,9 +31,9 @@ public function __invoke(Request $request, ModuleModel $model, string $section,
             return new Response('');
         }
 
-        $modules = StringUtil::deserialize($model->rootPageDependentModules);
+        $modules = array_filter(StringUtil::deserialize($model->rootPageDependentModules, true));
 
-        if (empty($modules) || !\is_array($modules) || !\array_key_exists($pageModel->rootId, $modules)) {
+        if (empty($modules) || !\array_key_exists($pageModel->rootId, $modules)) {
```

Comment:
Wouldn‘t it be much easier to replace the `!\array_key_exists()` check with an `empty()` check?

Reply:
Done.